src/Toolkit/tests/Fixtures/Kernel.php

```diff
@@ -16,21 +16,34 @@
 use Symfony\Bundle\TwigBundle\TwigBundle;
 use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
 use Symfony\Component\HttpKernel\Kernel as BaseKernel;
+use Symfony\UX\Icons\UXIconsBundle;
 use Symfony\UX\Toolkit\UXToolkitBundle;
 use Symfony\UX\TwigComponent\TwigComponentBundle;
+use TalesFromADev\Twig\Extra\Tailwind\Bridge\Symfony\Bundle\TalesFromADevTwigExtraTailwindBundle;
+use Twig\Extra\TwigExtraBundle\TwigExtraBundle;
 
 final class Kernel extends BaseKernel
 {
     use MicroKernelTrait;
 
     public function registerBundles(): iterable
     {
-        return [
+        $bundles = [
             new FrameworkBundle(),
             new TwigBundle(),
             new TwigComponentBundle(),
+            new TwigExtraBundle(),
+            new UXIconsBundle(),
             new UXToolkitBundle(),
         ];
+
+        if (class_exists(TalesFromADevTwigExtraTailwindBundle::class)) {
+            $bundles[] = new TalesFromADevTwigExtraTailwindBundle();
+        } elseif (\PHP_VERSION_ID >= 80200) {
+            throw new \RuntimeException('The dependency "tales-from-a-dev/twig-tailwind-extra" must be installed when using PHP 8.2+.');
+        }
```

Comment:
I want to force `tales-from-a-dev/twig-tailwind-extra` installation if we are using PHP 8.2+, in order to not forgot it when running tests